src/LightPanels/LightPanels.js

```diff
@@ -862,6 +867,13 @@ module.exports = kind(
 		for (var idx = 0; idx < viewProps.length; idx++) {
 			this.removeTask(this.getViewId(viewProps[idx]));
 		}
+	},
+
+	/**
+	* @private
+	*/
+	setupFallback: function () {
+		this._fallbackTimeout = global.setTimeout(this.bindSafely('transitionFinished'), this.duration + 100);
```

Comment:
It probably isn't absolutely necessary, but I wanted to have some safeguard so that the clean-up logic is not executed too early. Some of this logic hides the transitioning-out panel, so there could be a minor visual blip. Also, we are effectively using a "mask" to cover the transitioning panels, to prevent user clicks, and this is removed in our clean-up logic, so there is a (minor, though not entirely implausible due to QA practices) risk that something gets triggered/activated during transition by the user. I will say that `100ms` is slightly arbitrary, though. I'm open to something more definitive, or if you feel that the risk of these occurrences (and the potential impact) is low enough to do away with the offset.